pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -119,9 +119,18 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 			createRequest.Scheme = aws.String("internal")
 		}
 
+		var allocationIDs []string
+
+		if eipList, present := annotations["ServiceAnnotationLoadBalancerEIPAllocations"]; present {
```

Comment:
Did we mean `annotations[ServiceAnnotationLoadBalancerEIPAllocations]`?

Reply:
Correct. Corrected in 99a6ea0322.